src/AwsServiceProvider.php

```diff
@@ -26,7 +26,8 @@ class AwsServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $source = realpath(__DIR__ . '/../config/aws.php');
+        $source = __DIR__.'/../config/aws.php';
```

Comment:
No need to mess about with `DIRECTORY_SEPARATOR` tbh. PHP internally normalizes it when it's used in php's internal functions.